build/openresty/patches/LuaJIT-2.1-20210510_01-ffi-arm64-macos-fix-vararg-call-handling.patch

```diff
@@ -0,0 +1,62 @@
+From 521b367567dc5d91d7f9ae29c257998953e24e53 Mon Sep 17 00:00:00 2001
```

Comment:
Seems like we accidentally introduced those patches

Reply:
Yes, besides that, there are two patches related to M1 that I think can also be removed.

build/openresty/patches/lua-resty-core-0.1.22_03-make-resty.core.shdict-compatible-with-m1.patch
build/openresty/patches/lua-resty-core-0.1.22_04-make-resty.core.response-compatible-with-m1.patch

Reply:
I opened a PR (#13279 ) to remove them.

Reply:
2.8.4 (also 2.8.3) were built using kbt 4.25.5 and these patch files were in https://github.com/Kong/kong-build-tools/tree/4.25.5/openresty-patches/patches/1.19.9.1 . During the bazel migration we've migrated them into the kong repository, so they're not new patches.

I've diffed this PR's patch files and the kbt 4.25.5 patches and the only differences are hash fix plus one build config change. The build config change is needed by bazel build:

<img width="1610" alt="image" src="https://github.com/Kong/kong/assets/1159640/02a39acc-366d-4b9f-88ed-b51e55ce4940">